Steps for deleting the incorrectly created `kubernetes` LB and public IP (#3252)
spectrocloud/librarium: docs/docs-content/troubleshooting/nodes.md

For detailed information, review the cluster upgrades [page](../clusters/clusters.md).

## Clusters

## Scenario - Incorrect LoadBalancer `kubernetes` Created by Azure Cloud Controller Manager

> The `cloud-controller-manager` introduced in Azure Kubernetes v.1.29.x can make your existing external service IP

Reviewer: ```suggestion
The `cloud-controller-manager` introduced in Kubernetes v.1.29.x can make your existing external service IP
```

Reviewer: I _think_ this was introduced in the Kubernetes pack, not Azure specific right?

Author: No, as far as I know this is Azure-specific 🤔

> inaccessible. This issue occurs because `cloud-controller-manager` creates a default load balancer named `kubernetes`

Reviewer: ```suggestion
inaccessible. In Azure, this issue occurs because `cloud-controller-manager` creates a default load balancer named `kubernetes`
```

and an associated public IP, which conflicts with the existing network configuration.

While this issue has been resolved for Palette-managed Azure clusters, you need to manually delete the incorrectly

> created `kubernetes` load balancer and the associated public IP address from your Azure account.

Reviewer: ```suggestion
created `kubernetes` load balancer and the associated public IP address from the Azure resource group your cluster was deployed to.
```

### Debug Steps

> 1. Open a terminal session and use the following command to log in to your Azure account.

Reviewer: ```suggestion
1. Open a terminal session and use the following command to authenticate your Azure CLI session.
```

   ```shell
   az login
   ```

> 2. Use the following command to list load balancers in your resource group and locate the `kubernetes` load balancer.

Reviewer: ```suggestion
2. Use the following command to list load balancers in the resource group you deployed your cluster to and locate the `kubernetes` load balancer.
```

Reviewer: Perhaps some guidance here on which rg?

   ```

3. Use the following command to delete the `kubernetes` load balancer from your resource group.

   ```shell
   az network lb delete --name kubernetes --resource-group <resource-group>
   ```

4. Use the following command to list public IP addresses in your resource group and locate the `kubernetes-<hash>` load
   balancer.

   ```shell
   az network public-ip list --resource-group <resource-group> --output table
   ```

   ```shell
   Name                                         ResourceGroup      Location    Zones    Address        IdleTimeoutInMinutes    ProvisioningState
   -------------------------------------------  -----------------  ----------  -------  -------------  ----------------------  -------------------
   pip-docs-azure-k8s-1-29-apiserver            palette-tutorials  eastus      123      48.217.216.43  4                       Succeeded
   pip-docs-azure-k8s-1-29-node-outbound        palette-tutorials  eastus      312      48.217.222.29  4                       Succeeded
   //highlight-next-line
   kubernetes-a98181bf0e90b4425b80d11c21ba766f  palette-tutorials  eastus      231      4.255.120.41   4                       Succeeded
   ```

> 5. Use the following command to delete the `kubernetes-<hash>` public IP from your resource group.

Reviewer: Any way we can verify this? Perhaps repeat step 2 to make sure the deletion is successful?
